src/pages/ProfileForm/Step1Form/Step1FormHeader.tsx

```diff
@@ -12,7 +12,7 @@ function Step1FormHeader(): JSX.Element {
       <TextIntroduction
         className='mb-[3.75rem]'
         heading='Complete your user profile'
-        subheading='Please complete the fields below and select the financial institution you are authorized to file for. Once you have successfully associated your user profile with a financial institution you will have access to the filing platform and can begin the filing process.'
+        subheading='Complete the fields below and select the financial institution you are authorized to file for. Once you have successfully associated your user profile with a financial institution you will have access to the filing platform and can begin the filing process.'
```

Comment:
Change "filing platform" to "platform"

Reply:
Updated!

https://github.com/cfpb/sbl-frontend/pull/206/commits/a9da76dc05ca1a76e0e12a2bd65ed97444a16d12